torch_xla/experimental/fori_loop.py

```diff
@@ -12,40 +12,70 @@
 from torch._higher_order_ops.while_loop import while_loop_op
 
 
+def fori_loop(lower, upper, body_fun, one_value, init_val):
+
+  device = xm.xla_device()
+
+  def cond_fn(upper, lower, x):
+    return lower[0] < upper[0]
+
+  def body_fn(upper, lower, x):
+    one_value = torch.ones(1, dtype=torch.int32, device=device)
+    return (torch.sub(upper, one_value), lower, body_fun(one_value, x))
```

Comment:
`body_fun(one_value, x)` is this general enough to support any `user_body_func`?

Reply:
Per our discussion, @ManfeiBai to follow-up to make this API be more generic to work with any body callable?

Reply:
Thanks, updated to more generic body function